src/libs/actions/Task.ts

```diff
@@ -703,7 +704,7 @@ function setAssigneeValue(
 
         // If there is no share destination set, automatically set it to the assignee chat report
         // This allows for a much quicker process when creating a new task and is likely the desired share destination most times
-        if (!shareToReportID) {
+        if (!shareToReportID && !skipShareDestination) {
```

Comment:
I may be missing something here but as `shareToReportID` will be empty in our case, will this not work without the need for `skipShareDestination`?

Reply:
Ah, not quite. If `skipShareDestination` is set, we do NOT want to populate this, as it populates by default to the user's chat, which is not what we want in this flow